core/src/main/java/hudson/node_monitors/ResponseTimeMonitor.java

```diff
@@ -49,6 +48,9 @@
     public static final AbstractNodeMonitorDescriptor<Data> DESCRIPTOR = new AbstractAsyncNodeMonitorDescriptor<Data>() {
         @Override
         protected Callable<Data,IOException> createCallable(Computer c) {
+            if (c.isOffline()) {
```

Comment:
The original JIRA suggests not wanting to do this for agents that are marked offline manually too - at one point, I had a check here to make sure the `OfflineCause` wasn't `ResponseTimeMonitor.Data`. Think that's reasonable?